lib/bemxjst/index.js

```diff
@@ -519,7 +524,8 @@ BEMXJST.prototype.exportApply = function exportApply(exports) {
 
   // Add templates at run time
   exports.compile = function compile(templates) {
-    return self.compile(templates);
+    self.compile(templates);
+    return exports;
```

Comment:
Not sure it's a good behaviour at all, some times better to have immutable instances with templates.